manoseimas/mps_v2/static/components/filter_component.jsx

```diff
@@ -203,6 +228,53 @@ var SidebarFilter = React.createClass({
   }
 });
 
+var SidebarSubtabs = React.createClass({
+  componentDidMount: function() {
+    if (this.props.sticky_context){
+      $('.ui.sticky').sticky({
+        context: this.props.sticky_context,
+        offset: 70
+      });
+    }
+  },
+
+  setSelected: function(key) {
+    this.props.callback('lobbyists', key);
+  },
+
+  render: function() {
+    return (
+      <div className="two wide column">
+        <div className="ui sticky">
+          <div className="subtabs">
+            {Object.keys(this.props.options).map( function(key) {
+              if (key === 'header') {
+                return(
+                  <div>
+                    <span className="header title">{this.props.options[key].name}</span>
```

Comment:
I think you can do `.map( function(key, value) {` and then use `value.name` instead of repeating `this.props.options[key]` everywhere.